src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.AddNewKeywordAction.cs

```diff
@@ -51,7 +53,8 @@ private SyntaxNode GetNewNode(Document document, SyntaxNode node, CancellationTo
                 var fieldDeclaration = node as FieldDeclarationSyntax;
                 if (fieldDeclaration != null)
                 {
-                    newNode = fieldDeclaration.AddModifiers(SyntaxFactory.Token(SyntaxKind.NewKeyword));
+                    var generator = _document.GetLanguageService<SyntaxGenerator>();
```

Comment:
:bulb: I would suggest using `SyntaxGenerator.GetGenerator(_document)`, since that's part of the public API and makes for a better example of what other code fix authors can do. With that said, the form you used here is commonly seen in Roslyn's current code fixes. :smile: